### PR TITLE
Integration with spm batch

### PR DESCRIPTION
**Pull request by TomMaullin**
This PR integrates the viewer with SPM, thus addressing issue #27 . Full details of how this is done can be found in the readme.md file.

@cmaumet, Would you mind having a look at this PR and giving some feedback on the user interface when you are back please?

### CODE REVIEW

File: Data/jsons/pain_01.json [hunk 1]
```diff
@@ -0,0 +1,1554 @@
+{
```
Comment (cmaumet) [inline Data/jsons/pain_01.json]: @TomMaullin: could we remove this file from the codebase?

File: README.md [hunk 2]
```diff
@@ -4,47 +4,59 @@ Read and display NIDM-Results packs in the SPM environment.
...
-This should create output like the below.
+<img src="Doc/examplemenu2.png" width="200">  |
+:-------------------------:
+ **Fig. 2.** The SPM option in the batch window. |
+4. Enter the NIDM-Results file you wish to view and where you wish the output html to be stored.
+
+<img src="Doc/example_batch.png" width="200"> |  
+:-------------------------:|
+ **Fig. 3.** The NIDM-Results batch window. |
+5. If there are multiple excursion sets stored in the NIDM-Results file the contrast window will open. Select the contrasts you wish to view and click done.
```
Comment (cmaumet) [inline README.md]: Could we have the option to specify the contrast from the original batch rather than interactively opening the contrast manager? This would be similar to the behavior of 'SPM > Stats > Results Report' and would allow to run the NIDM-Results viewer completely in batch mode.

File: lib/changeNIDMtoSPM.m [hunk 3]
```diff
@@ -66,6 +66,24 @@
         contrastWeightMatrix = relevantToExcursion(contrastWeightMatrix, exID, exLabels);
     end
     
+    %Get the StatType and statisticMaps
```
Comment (cmaumet) [inline lib/changeNIDMtoSPM.m]: These changes do not seem to be related to the SPM batch integration. Can you shortly comment on why they appear here? E.g. do they address another issue, is it a bug fix, other?

File: lib/changeNIDMtoSPM.m [hunk 4]
```diff
@@ -78,6 +96,19 @@
     [~, name, ext] = fileparts(designMatrixFilename);
     xXtemp.xKXs.X = csvread(fullfile(filepathTemp, [name, ext]));
     
+    %Record the SVD data.
```
Comment (cmaumet) [inline lib/changeNIDMtoSPM.m]: Same comment as above. Can you (very) shortly explain why those changes were made?

Comment (TomMaullin) [reply lib/changeNIDMtoSPM.m]: If I am completely honest, I am not sure why I made these changes on this branch. I think I must have thought I was on a different branch. They are useful small changes though in that they result in the SPM object generated by changeNIDMtoSPM more closely resembling that generated by normal SPM output. I'd still recommend keeping them!

**Comment by TomMaullin** [comment]
Hi @cmaumet , I have now addressed all feedback. As it has been a while since we last looked at this PR I thought I'd best confirm, are you still happy for me to merge?

File: lib/changeNIDMtoSPM.m (hunk 4 above)
Comment (cmaumet) [reply lib/changeNIDMtoSPM.m]: Okay! That's fine.

**Comment by cmaumet** [comment]
@TomMaullin: This looks good to me. Can you check that the tests pass and copy/paste the test results in a comment below? 

Also, note that you latest commits are not associated with your GitHub profile, you might need to reset your email address in the git configurations: cf. https://help.github.com/articles/setting-your-commit-email-address-in-git/. (That's not a big deal for the commits above but it is worth setting it up now for future commits).

**Comment by TomMaullin** [comment]
Hi @cmaumet , I can confirm all tests pass. Here are the results:

result = 

  1×32 TestResult array with properties:

    Name
    Passed
    Failed
    Incomplete
    Duration
    Details

Totals:
   32 Passed, 0 Failed, 0 Incomplete.
   212.8268 seconds testing time.

Ah okay I will look into the email I've used.

**Comment by cmaumet** [comment]
Excellent! Thanks! I will let you merge :)
